python/test/glome_test.py

```diff
@@ -13,61 +13,44 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """
-Module that implements unittests cases for Glome Module.
+Module that implements unittests cases for Glome Class.
 """
 
 import unittest
 from cryptography.hazmat.primitives.asymmetric import x25519
 
 import pyglome
+from test import test_vectors
```

Comment:
I'm not sure whether referencing to the test subdirectory as a module is a good idea. Could you just use `import .test_vectors` isntead?

Reply:
I did this base on the [styleguide](https://google.github.io/styleguide/pyguide.html#233-decision). Maybe doesnt apply here because the test folder does not plan to be a package but some executables with a common background?

Reply:
That same section also asserts:

> This being the case, code should assume that import jodie refers to a third party or top level package named jodie, not a local jodie.py.

Imho the intention of the style guide is to enforce an association of imports with packages that are easily identified by a reader. I would not count the `test` directory as a top level / third party package in that sense, but I see that this might be ambiguous. If the tests were part of the top level glome package, there would at least be precedence: https://github.com/abseil/abseil-py/blob/master/absl/tests/app_test.py#L36. Not that I would recommend that, either.

Reply:
That being said, if this structure with a top level `test` module is useful for organizing the tests, please do keep it.

Reply:
I find quite useful that test is a top level module because it is easy to to and esay to understand  what is going on for a reader. It might be hepful as more files are added to the folder or even if the folder is subdivided in some subfolders in the future as the test are going to be easily moved around without any problems.